repo/change_password.go

```diff
@@ -28,18 +32,36 @@ func (r *directRepository) ChangePassword(ctx context.Context, newPassword strin
 
 	r.formatEncryptionKey = newFormatEncryptionKey
 
-	if err := encryptFormatBytes(f, repoConfig, newFormatEncryptionKey, f.UniqueID); err != nil {
+	if err = encryptFormatBytes(f, repoConfig, newFormatEncryptionKey, f.UniqueID); err != nil {
```

Comment:
Why is this change needed? the scope of `err` here can remain local to the block.

Reply:
I think this was due to some tests triggering on it, I'll attempt reverting.